src/flexalloc_mm.c

```diff
@@ -734,6 +734,11 @@ fla_print_pool_entry(struct flexalloc *fs, struct fla_pool_entry * pool_entry)
   uint32_t tmp;
   fprintf(stderr, "===============\n");
   fprintf(stderr, "Pool Entry %p\n", pool_entry);
+  fprintf(stderr, "obj_nlb : %"PRIu32"\n", pool_entry->obj_nlb);
```

Comment:
Not for this PR, but I'd like for functions like these to get removed in favor of some code in the python bindings that lists/dumps debug information.

Reply:
I actually like these ;). I use them a lot to debug. sometimes I place print statements before and after a call to see how they change. Like in the case of this PR, I printed it in the daemon and then I printed it in the client.
I have a question though:
Is there a way to inject a "print pool_entry" call from python into any place in the C code?